ja/lessons/basics/documentation.md

```diff
@@ -102,11 +102,11 @@ iex> h Greeter.hello
 
                 def hello(name)
 
-Hello メッセージを表示します
+Helloメッセージを表示します
 
 パラメータ
 
-  • name: 人名を表現する文字です。
+  •name: 人名を表現する文字です。
```

Comment:
```suggestion
  • name: 人名を表現する文字です。
```

Reply:
@marocchino It is better to put whitespace 👍